cassdegrees/templates/createprogram.html

```diff
@@ -1,44 +1,45 @@
 {% extends "base.html" %}
+{% load static %}
 
 {% block page-title %}Create Program Template{% endblock %}
 
 {% block subtitle %}Create Program Template{% endblock %}
 
 {% block content %}
     {% if msg != None %}
-        <p class="{% if is_error %}msg-error{% else %}msg-success{% endif %}">{{ msg }}</p>
+        <p class="{% if is_error %}msg-error{% else %}msg-success{% endif %}" id="serverError">{{ msg }}</p>
     {% endif %}
 
 
-    <form class="anuform" action="" method="post">
+    <form class="anuform" id="mainForm" action="" novalidate method="post">
         {% csrf_token %}
 
         <fieldset>
             <legend>Configure Program Details</legend>
 
-            <p>
+            <p class="form-group">
                 <label for="code">Program Code:</label>
-                <input class="text" id="code" type="text" name="code" aria-required="true" placeholder="e.g. BARTS" required value="{{ code|default:"" }}">
+                <input class="text" id="code" type="text" name="code" aria-required="true" minlength="4" placeholder="e.g. BARTS" required value="{{ code|default:"" }}">
             </p>
 
-            <p>
+            <p class="form-group">
                 <label for="name">Program Name:</label>
-                <input class="text" id="name" type="text" name="name" aria-required="true" placeholder="e.g. Bachelor of Arts" required value="{{ name|default:"" }}">
+                <input class="text" id="name" type="text" name="name" aria-required="true" minlength="5" placeholder="e.g. Bachelor of Arts" required value="{{ name|default:"" }}">
             </p>
 
-            <p>
+            <p class="form-group">
                 <label for="year">Program Year:</label>
                 <input class="text" id="year" type="number" min="2000" max="2100" value="{{ year|default:"2019" }}" name="year" aria-required="true" required>
             </p>
 
-            <p>
+            <p class="form-group">
                 <label for="units">Program Units:</label>
                 <input class="text" id="units" type="number" min="0" max="1000" value="{{ units|default:"144" }}" name="units" aria-required="true" required>
             </p>
 
-            <p>
+            <p class="form-group">
                 <label for="degreeType">Program Type:</label>
-                <select id="degreeType" name="degreeType" aria-required="true" required value="{{ degreeType|default:"" }}>
+                <select id="degreeType" name="degreeType" aria-required="true" required value="{{ degreeType|default:"" }}">
                     <option value="ugrad-sing" selected>Undergraduate Single Pass Degree</option>
                     <option value="ugrad-doub">Undergraduate Flexible Double Degree</option>
                     <option value="hon">Honours Degree</option>
```

Comment:
```suggestion
                     <option value="hon" {% if degreeType == "hon" %}selected{% endif %}>Honours Degree</option>
```

Mark as selected if degreeType is hon